README.md

```diff
@@ -258,6 +258,19 @@ image via a runtime mount:
 docker run -v /some-path/your-application:/app launcher.gcr.io/google/jetty  
 ```
 
+### Enabling dry-run
+The image's default start command will first run the jetty start.jar as a --dry-run to generate the JVM
+start command before starting the jetty web server. If you wish to generate the start command in your Dockerfile
+rather than at container start-time, you can run the `/scripts/jetty/gen_start_command.sh` script to generate it 
+for you, i.e. 
+
+```bash
+RUN export JETTY_START_COMMAND=$(/scripts/jetty/gen_start_command.sh)
```

Comment:
`gen_start_command.sh` doesn't exist anymore and `JETTY_START_COMMAND` is not used